lighthouse-extension/app/src/lighthouse-ext-background.js

```diff
@@ -190,53 +151,67 @@ window.loadSettings = function() {
 
       const defaultSettings = {
         useDevTools: false,
-        disableExtensions: disableExtensionsDuringRun,
       };
       const savedSettings = Object.assign(defaultSettings, result[SETTINGS_KEY]);
 
       resolve({
-        useDevTools: savedSettings.useDevTools,
+        useDevTools: !!savedSettings.useDevTools,
         selectedCategories: Object.keys(savedCategories).filter(cat => savedCategories[cat]),
-        disableExtensions: savedSettings.disableExtensions,
       });
     });
   });
-};
+}
 
-window.listenForStatus = function(callback) {
+/** @param {(status: [string, string, string]) => void} listenCallback */
+function listenForStatus(listenCallback) {
   log.events.addListener('status', function(log) {
     latestStatusLog = log;
-    callback(log);
+    listenCallback(log);
   });
 
   // Show latest saved status log to give immediate feedback
   // when reopening the popup message when lighthouse is running
   if (lighthouseIsRunning && latestStatusLog) {
-    callback(latestStatusLog);
+    listenCallback(latestStatusLog);
   }
-};
+}
 
-window.isRunning = function() {
+function isRunning() {
   return lighthouseIsRunning;
-};
+}
 
 // Run when in extension context, but not in devtools.
-if (window.chrome && chrome.runtime) {
-  // Get list of installed extensions that are enabled and can be disabled.
-  // Extensions are not allowed to be disabled if they are under an admin policy.
-  // chrome.management.getAll(installs => {
-  //   chrome.management.getSelf(lighthouseCrxInfo => {
-  //     installedExtensions = installs.filter(info => {
-  //       return info.id !== lighthouseCrxInfo.id && info.type === 'extension' &&
-  //              info.enabled && info.mayDisable;
-  //     });
-  //   });
-  // });
-
+if ('chrome' in window && chrome.runtime) {
   chrome.runtime.onInstalled.addListener(details => {
     if (details.previousVersion) {
       // eslint-disable-next-line no-console
       console.log('previousVersion', details.previousVersion);
     }
   });
 }
+
+if (typeof module !== 'undefined' && module.exports) {
+  // We don't want tsc to infer an index type, so use exports intest of module.exports.
```

Comment:
who consumes this via `require`?

Reply:
> who consumes this via require?

popup.js imports this for a typedef only. Added a comment to explain

Reply:
oh just for typedefs gotcha 👍  yeah comment is good